wagtail_streamfield_migration_toolkit/migrate_operation.py

```diff
@@ -67,34 +68,23 @@ def __init__(
     def migrate_stream_data_forward(self, apps, schema_editor):
         model = apps.get_model(self.app_name, self.model_name)
 
-        Revision = apps.get_model("wagtailcore", "Revision")
-        # We check if the models have a field `latest_revision` and make sure it points to the
-        # Revision model. This relation is there on models with `RevisionMixin`.
-        has_revisions = (
-            hasattr(model, "latest_revision")
-            and model.latest_revision.field.remote_field.model is Revision
-        )
-        # Again, check for `live_revision`
-        has_live_revisions = (
-            hasattr(model, "live_revision")
-            and model.live_revision.field.remote_field.model is Revision
-        )
+        if WAGTAIL_VERSION < (4, 0, 0):
```

Comment:
Hang on. We don't need the current wagtail version when we run this - we need the version that was used to generate this migration, the version at this point in the migration history. We need to use the `apps.get_model` to test which models we have to work this out.